change/@ni-nimble-blazor-2c03062d-afdf-4ad5-8aa9-08fb6714071f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Doing patch builds to test: 
- nothing breaks with deployment
- beachball will handle the duplicated peer / dev deps introduced